array-methods.js

```diff
@@ -21,7 +33,22 @@ var sumOfBankBalances = null;
   take each `amount` and add 18.9% interest to it rounded to the nearest dollar 
   and then sum it all up into one value saved to `sumOfInterests`
  */
-var sumOfInterests = null;
+
+var sumOfInterests = dataArray.filter(
+  function(element){
+    if (element.state === "WI" || element.state === "IL" || element.state === "WY" ||
+        element.state === "OH" || element.state === "GA" || element.state === "DE"){
+      return true;
+    } 
+}).map(
```

Comment:
The normal convention is to put the `.map` on the next line but indented in two spaces. This is for all chained methods except for the first one which can stay how it is right now for the `.filter`. Please make sure the rest of the file follows this convention.